symplyphysics/laws/hydro/volume_flux_is_constant.py

```diff
@@ -0,0 +1,43 @@
+from sympy import Eq, dsolve, Derivative
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    Function,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## The product of the area and the fluid speed, which is called volume flux, is constant 
+## at all points along the tube of flow of an incompressible liquid.
+
+# Law: d(A*v)/dt = 0
+## A - area of the tube of flow
+## v - fluid speed
+
+# It is also called the equation of continuity
+
+# Condition
+## The fluid should be ideal, i.e.
+## 1) nonviscous,
+## 2) in steady (laminar) flow,
+## 3) incompressible,
+## 4) irrotational.
+
+time = Symbol("time", units.time)
+tube_area = Function("tube_area", units.area)
+fluid_speed = Function("fluid_speed", units.velocity)
+
+law = Eq(Derivative(tube_area(time) * fluid_speed(time), time), 0)
+
+
+@validate_input(tube_area_before_=tube_area, fluid_speed_before_=fluid_speed, tube_area_after_=tube_area)
+@validate_output(units.velocity)
+def calculate_flow_speed(tube_area_before_: Quantity, fluid_speed_before_: Quantity, tube_area_after_: Quantity) -> Quantity:
+    solved = dsolve(law, fluid_speed(time))
+    result_expr = solved.subs({
+        "C1": tube_area_before_ * fluid_speed_before_,
```

Comment:
We should explain, where it came from. I suggest:
1. Find solution for differential equation: `dsolved = dsolve(law, fluid_speed(time))`
2. C1 is the same for time_before and time_after.
3. Find C1: `c1_value = solve(dsolved, "C1")[0].subs({tube_area(time): tube_area_before, fluid_speed(time): fluid_speed_before})`
4. Find resulting velocity: `result_expr = dsolved.subs({"C1": c1_value, tube_area(time): tube_area_after_})`